### DISCUSSION TITLE
How to select initramfs?

### DISCUSSION THREAD

**Discussion by aksdb** [zbm-dev/zfsbootmenu#652] [Questions] [answered]
I tried to understand from the docu, but I simply can't figure out how I can select a different initramfs during boot. ZBM seems to do some best-guess selection that is fine for default, but what if I need to select a different one? Archlinux for example generates a "normal" initramfs but also a "fallback" one with more modules in it that can be helpful when hardware has changed. How would I go about loading the other fallback one? Or a completely different one? (From a backup, or when playing with multiple initram-generators.)

**Comment by classabbyamp** [comment]
It's not currently possible to select

**Comment by zdykstra** [comment, marked as answer]
If you have one kernel and multiple initramfs files that map to it, your best option will be to create symlinks to the base kernel image that line up with the names of the initramfs files. This will then populate the kernel selection screen with entries for everything. You can try it by jumping into a chroot of the environment, creating the symlinks and then exiting back out. The kernel selection screen will repopulate with everything it finds.

**Reply by aksdb** [reply]
That's a neat idea. At least for my usecase (Arch's kernel images have static names - no versions included), that should solve it reasonably well. Thank you!
